### PR TITLE
Upgrade to the lates basement

### PR DESCRIPTION
**Pull request by Ferruck**

### CODE REVIEW

File: recipes/demo/freertos.yaml [hunk 1]
```diff
@@ -1,4 +1,12 @@
+inherit: [make]
+
 depends:
+    # tools
+    -
+        name: devel::make
+        use: [tools]
+        forward: True
+
```
Comment (jkloetzke) [inline recipes/demo/freertos.yaml]: This should not be required because the basement::rootrecipe class does already provide the make tool...

Comment (Ferruck) [reply recipes/demo/freertos.yaml]: It was indeed not needed. I don't know why I've added it in the first place. :confused: Sorry for the noise.

Comment (jkloetzke) [reply recipes/demo/freertos.yaml]: No problem. :)

**Comment by jkloetzke** [comment]
Thanks!

**Pull request merged**